components/layout/app-sidebar-client.tsx

```diff
@@ -168,7 +169,7 @@ export default function AppSidebarClient({
                         alt={session?.user?.name || ''}
                       /> */}
                       <AvatarFallback className="rounded-lg">
-                        {extractInitials(session?.user?.fullName ?? '')}
+                        {extractInitials(session?.user.fullName ?? '')}
```

Comment:
please revert back the nullish check. the reason i keep nullish check on data coming from the server is sometimes they are not predictable. A proper nullish check prevents us from having "undefined" values in out code